lib/internal/url.js

```diff
@@ -343,7 +343,7 @@ class URL {
     }
 
     if (typeof depth === 'number' && depth < 0)
-      return opts.stylize('[Object]', 'special');
+      return this;
 
```

Comment:
☝️ are these cool not being styled (or is this like the default styling?)

Reply:
This is the default styling and actually improves the output (it results in `[URL]`).